api/lucy-bot.ts

```diff
@@ -48,7 +44,7 @@ if (!PROD_ENV) {
   bot.launch();
 }
 
-module.exports = (req: NowRequest, resp: NowResponse) => {
+module.exports = async (req: NowRequest, resp: NowResponse): Promise<NowResponse> => {
   if (req.method === 'POST') bot.handleUpdate(req.body, resp);
-  else resp.status(200).send('Use POST to use Telegram bot!');
+  return resp.status(200).send('Use POST to use Telegram bot!');
```

Comment:
Is this change required?

Reply:
just additional type checking can be removed

Reply:
Okay. then

Reply:
Not resolved?